README.md

```diff
@@ -51,3 +51,9 @@ Guides are available on [HashiCorp Learn](https://learn.hashicorp.com/nomad).
 Contributing
 --------------------
 See the [`contributing`](contributing/) directory for more developer documentation.
+
+End-User License Agreement & Terms of Evaluation (Nomad Enterprise)
+--------------------------------------------------------------------
+The following shall apply unless your organization has a separately signed Enterprise License Agreement or Evaluation Agreement governing your use of the software:
+
+Software in this repository is subject to the license terms located in the software, copies of which are also available at [EULA](https://eula.hashicorp.com/ClickThruELA-Global.pdf) or [Terms Of Evaluation](https://www.hashicorp.com/terms-of-evaluation) as applicable. Please read the license terms prior to using the software. Your installation and use of the software constitutes your acceptance of these terms. If you do not accept the terms, do not use the software.
```

Comment:
I'm confused by the language here: The header and EULA links imply this only applies to Nomad Enterprise, but the text uses `this repository`, the OSS version.

Can we update the language to make the impact on the OSS community more explicit?

Reply:
Hi there @notnoop, you make very a valid statement! I just recently chatted with my team and we came to the conclusion that we should not update the OSS repos with the EULA + TOE blurb as it isn't one of the requirements explicitly called in our PRD, it was a confusion on my part. I will be closing this PR and will be going along my way! 😅 Thank you for taking the time to review this.

Reply:
No problem. Thanks for the context @claire-labry !